src/db/DB_Table.h

```diff
@@ -35,6 +35,8 @@ using namespace rapidjson;
 #include "html_template.h"
 using namespace tmpl;
 
+typedef wxLongLong int64;
```

Comment:
Not that it really matters, but should we be using wxLongLong as this is indicated as obsolete.  https://docs.wxwidgets.org/3.2/classwx_long_long.html

Reply:
Have built on MacOS and confirm it builds and runs fine.

Reply:
Thanks for trying. 

a major reason is wxSQLite3 still uses this in its GetInt64 

https://utelle.github.io/wxsqlite3/docs/html/classwx_s_q_lite3_result_set.html#a4b80559aac5a6d78a965aa9580d53684

Reply:
we may consider the switch along with wxSQLite3

@vomikan , @n-stein , can you help on win and linux platforms?

Reply:
+ @georgeef and @wolfsolver as FYI 
then all mobile and desktop mmex would work on large IDs